data/pilots/separatist-alliance/firespray-class-patrol-craft.json

```diff
@@ -95,7 +95,7 @@
     },
     {
       "name": "Boba Fett",
-      "xws": "bobafett-separatistalliance",
+      "xws": "bobafett-firesprayclasspatrolcraft",
```

Comment:
We don't want the faction name here?

Reply:
we talked in slack but in case it's useful for others, the first available id matches `pilotname-shipname`, which happens to be massively not helpful in this case. Same for Boba Fett Sep pilot, sadly.